src/coreclr/src/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -353,7 +353,7 @@ public static long GetTotalMemory(bool forceFullCollection)
         }
 
         [DllImport(RuntimeHelpers.QCall, CharSet = CharSet.Unicode)]
```

Comment:
I was considering removing `CharSet = CharSet.Unicode` here and in `_UnregisterFrozenSegment`, since these two methods don't marshal any strings. Thoughts?

Reply:
We use this on all QCalls, independent on whatever they marshal any strings or not.